recipes-devtools/ghc/files/ghc-pkg-wrapper

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+args=$@
+recache=0
+
+ghc_pkg() {
+    ${GHC_PKG_PATH}/ghc-pkg $@
```

Comment:
You always want `"$@"` so that white space splitting doesn't happen.  Or put another way to ensure args "a" "b c" "d" stay that way and don't turn into "a" "b" "c" "d"